### PR TITLE
Opens PR when a new OpenJDK EA release comes out

### PR DESCRIPTION
**Pull request by zakkak**

### CODE REVIEW

**Comment by zakkak** [comment]
As indicated in #43 we need to keep up with the latest openJDK EA.

**Comment by zakkak** [comment]
@Karm I understand your confusion.
Note that these actions are not building anything they just keep branches in sync (they run daily).
Keeping them on master (regardless of what branch they operate on) makes life easier (less duplication, easier to copy-paste similar parts, etc.).
In theory we could (and maybe we should) have them in a totally different branch (e.g. something like `mandrel-sync-actions`).

File: .github/workflows/repo-sync.yml [hunk 1]
```diff
@@ -58,3 +58,29 @@ jobs:
...
+    - name: Checkout 20.1 branch
+      uses: actions/checkout@v2
+      with:
+        ref: 20.1
+        fetch-depth: 0
+    - name: Check and update OpenJDK version
+      run: |
+        git config --local user.email "fzakkak@redhat.com"
+        git config --local user.name "Foivos Zakkak"
+        curl https://api.github.com/repos/AdoptOpenJDK/openjdk11-upstream-binaries/releases | jq -r 'first | .assets[] | .browser_download_url' > dl_urls.txt
```
Comment (jerboaa) [inline .github/workflows/repo-sync.yml:74]: Would this API URL help by any chance?
```
$ curl -v https://api.adoptopenjdk.net/v3/binary/latest/11/ea/linux/x64/jdk/hotspot/normal/openjdk 2>&1 | grep 'location: '
< location: https://github.com/AdoptOpenJDK/openjdk11-upstream-binaries/releases/download/jdk-11.0.8%2B7/OpenJDK11U-jdk_x64_linux_11.0.8_7_ea.tar.gz
```
There is also https://api.adoptopenjdk.net/v3/binary/latest/11/ea/linux/x64/staticlibs/hotspot/normal/openjdk

for the static libs artifact.

Comment (zakkak) [reply .github/workflows/repo-sync.yml:74]: It removes the `jq` magic so it's probably easier to maintain, thanks!

**Pull request merged**